packages/axios-extension/src/abap/lrep-service.ts

```diff
@@ -256,6 +256,28 @@ export class LayeredRepositoryService extends Axios implements Service {
         }
     }
 
+    /**
```

Comment:
I don't think that one is a good idea. You are already in possession of the full url and replace parts in here because you know that the url will be going to this service. That is quite some implicit knowledge.

I'd recommend to remove this function and where you currently call it, just call `provider.get(manifestUrl)`

Reply:
I think that using directly `provider.get(manifestUrl)` would lead to a need of handling the errors in the CLI. There is also a possibility that we would need this method for the other generators. 
The replace is no longer needed with: a83e51f8dfe9d80500d2884300109b86c866e406

Reply:
Please revert a83e51f. The input parameter for the `getLayeredRepository` is there to define an alias if the service is available on a different url, but you misuse it for a hack.  If you call `getLayeredRepository('/')` then the returned service will not work with any of the other offered functions.

Reply:
Fixed with: c0d4b4265d63513c9a5639844b3d45ab8f1567a4